docs/Learning-Environment-Examples.md

```diff
@@ -354,35 +354,70 @@ you would like to contribute environments, please see our
 
 ![SoccerTwos](images/soccer.png)
 
-- Set-up: Environment where four agents compete in a 2 vs 2 toy soccer game.
-- Goal:
-  - Get the ball into the opponent's goal while preventing the ball from
-    entering own goal.
-- Agents: The environment contains four agents, with the same Behavior
-  Parameters : Soccer.
-- Agent Reward Function (dependent):
-  - +1 When ball enters opponent's goal.
-  - -1 When ball enters team's goal.
-  - -0.001 Existential penalty.
-- Behavior Parameters:
-  - Vector Observation space: 336 corresponding to 11 ray-casts forward
-    distributed over 120 degrees (264) and 3 ray-casts backward distributed over
-    90 degrees each detecting 6 possible object types, along with the object's
-    distance. The forward ray-casts contribute 264 state dimensions and backward
-    72 state dimensions.
-  - Vector Action space: (Discrete) Three branched actions corresponding to
-    forward, backward, sideways movement, as well as rotation.
-  - Visual Observations: None
-- Float Properties: Two
-  - ball_scale: Specifies the scale of the ball in the 3 dimensions (equal
-    across the three dimensions)
-    - Default: 7.5
-    - Recommended minimum: 4
-    - Recommended maximum: 10
-  - gravity: Magnitude of the gravity
-    - Default: 9.81
-    - Recommended minimum: 6
-    - Recommended maximum: 20
+* Set-up: Environment where four agents compete in a 2 vs 2 toy soccer game.
```

Comment:
Please be consistent with the rest of this document, it seems this one uses - instead of *

Reply:
I am shocked that I did that....